src/Library/DeviceBase.cpp

```diff
@@ -90,14 +90,20 @@ namespace usbguard
      * Set connect type
      */
     int retry = 0;
-    std::string connect_type;
-    while (retry < 20) {
+    std::string connect_type = "";
```

Comment:
Adding ` = ""` does not seem to change semantics or add value. I checked back with http://www.cplusplus.com/reference/string/string/string/. Let's drop that part for a cleaner diff, okay?